test/named_routes_test.rb

```diff
@@ -50,7 +50,7 @@
     end
 
     it 'recognizes glob string' do
-      @router.path(:glob).must_equal '/files/'
+      @router.path(:glob).must_equal '/files'
```

Comment:
that is because of this https://github.com/lotus/router/blob/master/lib/lotus/routing/http_router.rb#L48

maybe you can get rid that Utils::PathPrefix and modify this: 
https://github.com/lotus/router/blob/master/lib/lotus/routing/http_router.rb#L162
`options[:prefix] || ''`
using something like `@prefix.concat uri.path`